service/javadsl/api/src/main/java/com/lightbend/lagom/javadsl/api/transport/TransportException.java

```diff
@@ -41,13 +42,13 @@ public TransportException(TransportErrorCode errorCode, ExceptionMessage excepti
     }
 
     public static TransportException fromCodeAndMessage(TransportErrorCode errorCode, ExceptionMessage exceptionMessage) {
-        BiFunction<TransportErrorCode, ExceptionMessage, TransportException> creator = BY_NAME_TRANSPORT_EXCEPTIONS.get(exceptionMessage.name());
+        Function<ExceptionMessage, TransportException> creator = BY_NAME_TRANSPORT_EXCEPTIONS.get(exceptionMessage.name());
         if (creator != null) {
-            return creator.apply(errorCode, exceptionMessage);
+            return creator.apply(exceptionMessage);
```

Comment:
This changes behavior slightly. Before this change, a user could `throw BadRequest(419, "Unsupported Data/Bad Request")` and would get that back (the message matches and the error code on reconstruction is taken from the response. With this change, the `419` is lost.